src/shared/seccomp-util.c

```diff
@@ -680,7 +695,7 @@ const SyscallFilterSet syscall_filter_sets[_SYSCALL_FILTER_SET_MAX] = {
                 .name = "@reboot",
                 .help = "Reboot and reboot preparation/kexec",
                 .value =
-                "kexec\0"
+                "kexec_load\0"
```

Comment:
`kexec_load` should go after `kexec_file_load` to make `test-seccomp` pass.

Reply:
ouch, indeed